packages/user/src/views/Login.tsx

```diff
@@ -11,7 +11,7 @@ import login, { verifySession } from "@/supertokens/login";
 
 import { userContext } from "../context/UserProvider";
 
-import type { LoginCredentials, UserContextType } from "@/types";
+import type { LoginCredentials, UserContextType } from "@/types/types";
```

Comment:
@shailesh896 `types/types`: can we have `index.ts` inside `types` and just import `@/types`